### PR TITLE
Remove multi rule FileParsingRule

### PR DESCRIPTION
**Pull request by marschwar**
This is part of #5192.

All rules from the `FileParsingRule` multi rule are now run individually.

### CODE REVIEW

**Review by schalkms — APPROVED** [review]

**Review by cortinico — APPROVED** [review]

**Comment by schalkms** [comment]
@cortinico seems to be an issue with danger and the GitHub rate limit. Is there something to adapt in the setup?

```
Request failed [403]: https://api.github.com/repos/detekt/detekt/pulls/5193
[19](https://github.com/detekt/detekt/runs/7707582017?check_suite_focus=true#step:5:20)
Response: {
[20](https://github.com/detekt/detekt/runs/7707582017?check_suite_focus=true#step:5:21)
  "message": "API rate limit exceeded for installation ID 7815900.",
[21](https://github.com/detekt/detekt/runs/7707582017?check_suite_focus=true#step:5:22)
  "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"
[22](https://github.com/detekt/detekt/runs/7707582017?check_suite_focus=true#step:5:23)
}
```

**Comment by cortinico** [comment]
> Is there something to adapt in the setup?

Not really. The problem is with codecov though. For some reason it seems like they post a comment that they edit over and over:

Screenshot 2022-08-06 at 23 43 29

This is generating 200+ invocations of the Danger workflow causing the own Github token to go out of quota. The labeller also failed.
This should be reported to the Codecov App I believe.

I'm looking into potentially skipping the workflow if the event trigger was coming from Codecov

**Comment by schalkms** [comment]
Ouch. 🙈

> This is generating 200+ invocations of the Danger workflow causing the own Github token to go out of quota.

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/KtFileContent.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import org.jetbrains.kotlin.psi.KtFile
+
+internal data class KtFileContent(val file: KtFile, val content: Sequence<String>)
+
+internal fun KtFile.toFileContent() = KtFileContent(this, text.splitToSequence("\n"))
```
Comment (chao2zhang) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/KtFileContent.kt:7]: ❤️ for internal

**Review by chao2zhang — APPROVED** [review]

**Pull request merged**